extensions/commons/README.md

```diff
@@ -0,0 +1,130 @@
+# Commons Extension Spec (`c`)
```

Comment:
Why is there the prefix 'c' mentioned? The extension page says there's no prefix.

Reply:
ah, thanks, fixing...